gr-qtgui/include/gnuradio/qtgui/EyeDisplayPlot.h

```diff
@@ -55,7 +55,7 @@ class EyeDisplayPlot : public DisplayPlot
 
     const QColor getTagTextColor();
     const QColor getTagBackgroundColor();
-    const Qt::BrushStyle getTagBackgroundStyle();
+    Qt::BrushStyle getTagBackgroundStyle();
```

Comment:
`const float foobar() -> float foobar() const`

Reply:
Done. Also `getTagTextColor` and `getTagBackgroundColor`.